readmeio/api-explorer
# feature: prompt to log into docs for hubs with logs

## 🧰 What's being changed?

For hubs/explorers logs enabled, we'll encourage logged-out users to log into the hubs to see their logs:

![image](https://user-images.githubusercontent.com/8854718/81227176-98936000-8fb1-11ea-9037-631f6cf8d3b6.png)

## 🧪 Testing

I'll open up a PR in ReadMe, but to test in this repo:
1. Check out this branch
2. In in `Demo.jsx`, uncomment this line:
https://github.com/readmeio/api-explorer/blob/9af939466394d5ebeb62bea74eabbd89f4856e5a/example/src/Demo.jsx#L114
and comment out these lines:
https://github.com/readmeio/api-explorer/blob/9af939466394d5ebeb62bea74eabbd89f4856e5a/example/src/Demo.jsx#L124-L129
3. Run `npm start` from the root of this repo.

## 🗳 Checklist
> 💡 If answering yes to any of the following, include additional info, before/after links, screenshots, etc. where appropriate!

* [x] **🆕 I'm adding something new!**
* [ ] **🐛 I'm fixing a bug!**
* [ ] **📸 I've made some changes to the UI!**
